pyHalo/Halos/HaloModels/ULDM.py

```diff
@@ -86,9 +90,56 @@ def profile_args(self):
             truncation_radius = self._lens_cosmo.LOS_truncation_rN(self.mass, self.z,
                                                              self._args['LOS_truncation_factor'])
 
-            self._profile_args = (self.c, truncation_radius)
+            # using 'mass' as the ULDM virial mass
+            [theta_c, kappa_0] = self._uldm_args(self._args['m_uldm'], self.mass, self._args['uldm_plaw'])
+
+            self._profile_args = (self.c, truncation_radius, theta_c, kappa_0)
 
         return self._profile_args
+    
+    def _uldm_args(self, m, M, plaw):
```

Comment:
can you change the 'm_uldm' keyword to "log10_m_uldm"? I think a logarithmic scale is a more natural choice since we are mainly concerned with the order of magnitude of the particle mass